svgnative/CMakeLists.txt

```diff
@@ -297,9 +330,46 @@ endif()
 ##############################
 # Creating pkg-config file
 ##############################
+include(GNUInstallDirs)
 
-# TODO: Add includes and libraries for individual ports.
 set(BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}")
 set(SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
-set(PRIVATE_LIBS)
-CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib.pc.in" "${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib.pc" @ONLY)
+set(CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
+set(CMAKE_INSTALL_BINDIR "${CMAKE_INSTALL_BINDIR}")
+set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}")
+set(CMAKE_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}")
+set(PRIVATE_LIBS "${PRIVATE_LIBS}")
+set(REQUIRES "${REQUIRES}")
+set(PRIVATE_REQUIRES "${PRIVATE_REQUIRES}")
+set(PORTS_INCLUDES "${PORTS_INCLUDES}")
+set(HEADER_SUBDIR "svgnative")
+CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib.pc.in"
+               "${CMAKE_CURRENT_BINARY_DIR}/SVGNativeViewerLib.pc" @ONLY)
+CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib-uninstalled.pc.in"
+               "${CMAKE_CURRENT_BINARY_DIR}/SVGNativeViewerLib-uninstalled.pc" @ONLY)
```

Comment:
I've never seen a special 'uninstalled' version of a `.pc` file.  Is there another library that has something similar?

Reply:
Please check the cairo source. There are [cairo.pc.in](https://cgit.freedesktop.org/cairo/tree/src/cairo.pc.in) and [cairo-uninstalled.pc.in](https://cgit.freedesktop.org/cairo/tree/src/cairo-uninstalled.pc.in). You can find uninstalled pc file writes the build directory pathname, but regular pc file does not have such.

Reply:
@lemzwerg concerns with this still?